app/scripts/proactive/model/ControlFlow.js

```diff
@@ -0,0 +1,53 @@
+define(
+    [
+        'backbone',
+        'proactive/model/SchemaModel',
+        'text!proactive/templates/script-form-template.html',
+    ],
+
+    function (Backbone, SchemaModel, tpl) {
+
+    "use strict";
+
+    var scriptTemplate = _.template(tpl);
+
+    return SchemaModel.extend({
+        schema: {
+            "Script": {
+                type: "TextArea",
+                fieldAttrs: {
+                    'placeholder': ['code->#cdata-section', 'code->#text']},
+                    template: scriptTemplate
+            },
+            "Language": {
+                type: 'Select',
+                options: [" ", "bash", "cmd", "docker-compose", "groovy", "javascript", "python", "cpython", "ruby", "perl", "powershell", "R"],
```

Comment:
one question, why cpython is called cpython and jython is called python?

Reply:
because jython is supplied by jsr-223, and it is called python, we can not change it